sympy/physics/tests/test_units.py

```diff
@@ -20,6 +20,8 @@ def test_units():
     t = Symbol('t')
     assert integrate(t*m/s, (t, 1*s, 5*s)) == 12*m*s
     assert (t * m/s).integrate((t, 1*s, 5*s)) == 12*m*s
+    
+    assert (year / week).evalf(15) == Float(52.17745571428571)
```

Comment:
Even if this test is as you say, this served the purpose of teaching someone how to write unit test. It can also possibly catch regressions and it adds virtually no time to the testing routines. How about we leave it in primarily for the sake of learning?